src/script/main/app.js

```diff
@@ -145,7 +145,7 @@ class App {
     );
     repositories.lifecycle = new z.lifecycle.LifecycleRepository(this.service.lifecycle, repositories.user);
     repositories.connect = new z.connect.ConnectRepository(this.service.connect, repositories.properties);
-    repositories.links = new z.links.LinkPreviewRepository(this.service.asset, repositories.properties);
+    repositories.links = resolve(graph.LinkPreviewRepository);
```

Comment:
You can remove this from here and put the `resolve` right in the constructor of the `ConversationRepository` a few lines down